pages/CheckingPage.js

```diff
@@ -0,0 +1,66 @@
+import BasePage from '../pages/BasePage';
+
+class CheckingPage extends BasePage {
+
+    get standardCheckingButton (){return $('label[for="Standard Checking"]')}
+
+    get interestCheckingButton (){return $('label[for="Interest Checking"]')}
+  
+    get individualCheckingButton (){return $('#Individual')}
+  
+    get jointCheckingButton (){return $('#Joint')}
+  
+    get checkingNameInput (){return $('#name')}
+  
+    get checkingDepositInput (){return $('#openingBalance')}
+  
+    get newCheckingFormSubmitButton () {return $('#newCheckingSubmit')}
+
+    get newCheckingFormResetButton () {return $('#newCheckingReset')}
+
+    get newCheckingConfirmation () {return $('#new-account-conf-alert')}
+
+    get newCheckingErrorText () {return $('#new-account-error-msg')}
+
+    get switchChecking () { return $('.switch-label')}
```

Comment:
Every selector that points to something on the "form/card" that represents the summary of a specific checking account has to reference to the specific account it wants, as that representation is actually part of a collection of accounts that renders one per existing account (therefore, as you create accounts, you'll have "n" replicas of the form/card, so this class will be present in all of them).

Make sure to consider that scenario in your solution (crafting a selector that points to a specific account)

Reply:
I use the switch checking only when two cards are created, so by pressing the switch label it changes the card, the value of each card changes when I do the tests because I have to delete and create data.

Reply:
What I mean is that this class (switch-label) is present on each "card" representing an account, so it doesn't individualize a specific account, it'll grab the first one it encounters with that class. This binds it to the specific order structure the cards have. That's what makes it less robust. You can craft a selector that uniquely identifies a specific account that you want, independent of order, only based on account's attributes, using **xpath**. 

You can take a look at this topic for inspiration, where we solved a similar problem with xpath. Ref: https://abstracta.ryver.com/#posts/2957788

Reply:
How does the tree works when I want to get it as an element?